app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,60 @@
   </head>
 
   <body>
+
+ <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+
+    <a class="navbar-brand" href="http://nussucommit.com">
+      <img src="DSCF4500.jpg" width="70" height="70" alt="">NUSSU CommIT
+    </a>
+
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#NavbarContent" aria-controls="NavbarContent" aria-expanded="false" aria-label="Toggle navigation">
+    </button>
+
+    <div class="collapse navbar-collapse" id="NavbarContent">
```

Comment:
Change to `navbar-content` for the ID. And change all references of `NavbarContent`